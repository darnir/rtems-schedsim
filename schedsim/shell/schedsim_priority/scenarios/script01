echo "*** TEST 01 ***"
rtems_init
echo "=== Create and delete 0x0a010001 ==="
task_create TA1 3
task_delete TA1
echo "=== Create and delete 0x0a010002 ==="
task_create TA1 3
task_delete 0x0a010002
echo "*** END OF TEST 01 ***"
exit
# We will not get here
