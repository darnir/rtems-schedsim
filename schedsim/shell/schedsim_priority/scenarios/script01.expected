Script File               : script01.scen
verbose                   : 1
==> 1: echo "*** TEST 01 ***"
*** TEST 01 ***
==> 2: rtems_init
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
==> 3: echo "=== Create and delete 0x0a010001 ==="
=== Create and delete 0x0a010001 ===
==> 4: task_create TA1 3
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010001, priority=3
Task (TA1) starting: id=0x0a010001, priority=3
  CPU 0: Thread Heir: 0x0a010001 priority=3
  CPU 0: Thread Executing: 0x0a010001 priority=3
==> 5: task_delete TA1
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
Task (0x0a010001) deleted
==> 6: echo "=== Create and delete 0x0a010002 ==="
=== Create and delete 0x0a010002 ===
==> 7: task_create TA1 3
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010002, priority=3
Task (TA1) starting: id=0x0a010002, priority=3
  CPU 0: Thread Heir: 0x0a010002 priority=3
  CPU 0: Thread Executing: 0x0a010002 priority=3
==> 8: task_delete 0x0a010002
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
Task (0x0a010002) deleted
==> 9: echo "*** END OF TEST 01 ***"
*** END OF TEST 01 ***
==> 10: exit
